config/optional/linkit.linkit_profile.default.yml

```diff
@@ -3,29 +3,30 @@ status: true
 dependencies:
   module:
     - node
-id: basic_html
+id: default
 label: 'Basic HTML'
```

Comment:
Label should be 'Default', otherwise its confusing when used in basic html and full html input filters